addons/web/static/lib/odoo_ui_icons/style.css

```diff
@@ -0,0 +1,341 @@
+@font-face {
+  font-family: 'odoo_ui_icons';
+  src: url('fonts/odoo_ui_icons.woff?wpto0f') format('woff');
```

Comment:
@stefanorigano Seems that we need a premium account on IconMoon to generate woff2 but there is not other way to convert it to woff2?

Reply:
https://github.com/google/woff2

Reply:
Just wanted to avoid adding yet another step for the future maintenance of these icons... 
I was wrong though, woff2 is definitively worth 👍